### ISSUE TITLE
Nothing fails when I tried something

### ISSUE THREAD

**Issue by TheAnalogyGuy** [kelseyhightower/nocode#5164]
I noticed a lot of others have tried `         ` and got `         `.. which is what I was trying to do, but when I tried something and got something instead of nothing, I think there may be a bug somewhere and this is not robust enough to handle anything. It should provide `         ` for anything, but that's not what happened. Something failed, but when I tried nothing, it succeeded, so there is a bug somewhere preventing something from providing `           ` and I can't figure out where it is.

Wondering if anyone else has experienced this. I am worried about nothing working properly so I opened this issue to investigate nothing and maybe see if there was anything we could do with nothing to improve it from behaving this way. I have been doing nothing for a while, and I would continue doing nothing, but then no one would see that the way to succeed was to do nothing and that sage advice would be lost if it wasn't documented somewhere. In case anyone else if having this issue, I figured out the issue - it was that I was trying something.

In summary, to succeed at nothing, you must not try something, or anything, for that matter, or you will fail at nothing. Please see the attachment for a screenshot of nothing succeeding.

**Comment by TheAnalogyGuy** [comment]
Nothing is working and should be good to close if nobody concurs. Nothing was done to fix it, so it is good to close.

**Comment by xuelink** [comment]
@TheAnalogyGuy Can you assign me that issue? I am planning to do nothing with this issue.
Big thanks!

**Comment by TheAnalogyGuy** [comment]
> @TheAnalogyGuy Can you assign me that issue? I am planning to do nothing with this issue. Big thanks!

Done.

**Comment by mohamedzhioua** [comment]
Could you please delegate that task to me? I have no intention of taking any action on it. Many thanks!

**Comment by mohamedzhioua** [comment]
@TheAnalogyGuy Could you please delegate that task to me? I have no intention of taking any action on it. Many thanks!

**Comment by MadTinker** [comment]
Please don't inform me if this changes.

**Comment by TheAnalogyGuy** [comment]
Nothing was successfully assigned. Nothing was completed. Nobody did anything. Nothing works as intended. Closing ticket as complete.
